File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.py
class Solution(object):
    def getMinDistance(self, nums, target, start):
        """
        :type nums: List[int]
        :type target: int
        :type start: int
        :rtype: int
        """

        minDiff = 100000

        for i in range( 0 , len(nums)):
            if( nums[i] == target and abs( start - i ) < minDiff ):
                minDiff = abs( start - i )

        return minDiff 
        

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        
        int minDiff = INT_MAX  ;

        for( int i = 0 ; i < nums.size() ; i++ ){
            if( nums[i] == target && abs( i - start ) <= minDiff ){
                minDiff = abs( i - start ) ;
            }
        }

        return minDiff ;
    }
};